package.json

```diff
@@ -74,9 +74,10 @@
     "@testing-library/user-event": "^11.0.1",
     "chalk": "4.1.0",
     "coveralls": "^3.0.6",
-    "enzyme": "^3.10.0",
```

Comment:
nit: the ending space is unnecessary.
I will open a PR soon to convert the rest of enzyme tests into RTL. So we can get rid of enzyme from dev dependency : )

Reply:
sweeeeeeet re enzyme! that will be amazing 🦾